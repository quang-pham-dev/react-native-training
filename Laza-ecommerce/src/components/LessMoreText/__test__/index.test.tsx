import React from 'react';
import renderer from 'react-test-renderer';

import MoreLessText from 'components/LessMoreText';

describe('Read more Component', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<MoreLessText numberOfLines={2} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
