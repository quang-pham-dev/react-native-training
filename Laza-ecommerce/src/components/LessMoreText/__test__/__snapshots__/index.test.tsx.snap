// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Read more Component should render correctly 1`] = `
<Text
  onTextLayout={[Function]}
  style={
    Array [
      Object {
        "fontFamily": "Inter_600SemiBold",
        "fontSize": 15,
        "lineHeight": 22,
      },
      undefined,
    ]
  }
/>
`;
